Tasks/AzureResourceGroupDeploymentV2/operations/ResourceGroup.ts

```diff
@@ -181,7 +181,20 @@ export class ResourceGroup {
                         policyLink = this.getPolicyHelpLink(error.details[i]);
                         errorMessage = this.getPolicyErrorMessage(error.details[i]);
                     } else {
-                        errorMessage = util.format("%s: %s %s", error.details[i].code, error.details[i].message, error.details[i].details);
+                        errorMessage = util.format("%s: %s", error.details[i].code, error.details[i].message);
+                        if(error.details[i].details != null && error.details[i].details != undefined) {
```

Comment:
Why is this condition required now? Is was not there earlier. was some issue reported due to this?

Reply:
If at all required, can't we just say 
```suggestion
                        if(error.details[i].details) {
```